### ISSUE TITLE
vignette or tutorial for workflow inbomd report -> Github release -> Zenodo archive

### ISSUE THREAD

**Issue by hansvancalster** [inbo/INBOmd#72]
For INBO reports published with INBOmd via gh-pages, it would be great to have a vignette or tutorial that documents the steps one has to take to:

1. link the GitHub repo to Zenodo 
2. create a GitHub release 
3. publish on gh-pages
4. add the zenodo archive to the inbo community on zenodo

I've created this issue here, but it could also be considered as a tutorial in https://github.com/inbo/tutorials

**Comment by florisvdh** [comment]
Good idea @hansvancalster ; I rather favour the idea of doing most of it in the tutorials website: the tutorial(s) would then focus on the github/zenodo aspects, which are not specific to INBOmd.

Probably such tutorial could be shorter and more generic if a separate "release=>Zenodo" tutorial is distinguished from a "gh-pages" one. I think publishing a website/report via inbo.github.io is independent of the release+Zenodo workflow. The Zenodo tutorial can still be referred from the gh-pages tutorial.

If you would like an all-in-one tutorial, specifically for INBO reports, then a vignette in INBOmd could add specific things and refer to the more generic tutorials with no need to repeat them.

**Comment by ThierryO** [comment]
I second @florisvdh that the tutorials website is more suitable.

IMHO every release requires an update of the ghpages branch. I'd only do more frequent ghpages updates in case we want to display public drafts of a report.
